### DISCUSSION TITLE
show partitions ($partitions)

### DISCUSSION THREAD

**Discussion by VanValdenburg** [aws/aws-sdk-pandas#1835] [Q&A] [answered]
AWS user guide provided the information how to shows partitions via SQL: https://docs.aws.amazon.com/athena/latest/ug/show-partitions.html
Can i achieve this in awswrangler?

I can execute this in the AWS console:
SELECT * FROM "database"."table$partitions"

But when i try the same in awswrangler i get an error:
An error occurred (InvalidRequestException) when calling the StartQueryExecution operation: line 5:36: mismatched input '$'. Expecting: ',', '.', 'AS', 'CROSS', 'EXCEPT', 'FOR', 'FULL', 'GROUP', 'HAVING', 'INNER', 'INTERSECT', 'JOIN', 'LEFT', 'LIMIT', 'NATURAL', 'OFFSET', 'ORDER', 'RIGHT', 'TABLESAMPLE', 'UNION', 'WHERE', 'WITH', <EOF>, <identifier>

I couldn't find anything in the awswrangler documentation on how to list partitions in any other way.

**Comment by jaidisido** [comment, marked as answer]
There are methods to list partitions already:
- https://aws-sdk-pandas.readthedocs.io/en/stable/stubs/awswrangler.catalog.get_partitions.html#awswrangler.catalog.get_partitions
- https://aws-sdk-pandas.readthedocs.io/en/stable/stubs/awswrangler.catalog.get_parquet_partitions.html#awswrangler.catalog.get_parquet_partitions
